index.js

```diff
@@ -10,8 +10,7 @@ const normalizeOptions = require("./src/options").normalize;
 const parser = require("./src/parser");
 const printDocToDebug = require("./src/doc-debug").printDocToDebug;
 const config = require("./src/resolve-config");
-
-const PRAGMA_RE = /@\bprettier|format\b/;
+const docblock = require("jest-docblock");
```

Comment:
My only concern on using `jest-docblock` for extracting the tags is that it has no documentation or information around it, (https://www.npmjs.com/package/jest-docblock).

If that is a multi-purpose module maybe we should include some documentation around it, otherwise we could either implement our own naive docblock pragma extractor or use other established parser like doctrine.

@cpojer what are your thoughts around it ? is  `jest-docblock`  meant to be a private module for jest itself ?

Reply:
it's already used to do the pragma extraction in `eslint-plugin-prettier` :)

it's so useful I'd be happy to document its api.

I imagine jest split itself as many packages for this very reason?

Reply:
`jest-docblock` in particular was split out because it was needed in multiple packages inside of jest - an awesome side-effect is that it can be used separately by other projects!

It has no documentation since I was lazy when it was extracted, so a documentation PR would be awesome

Reply:
Use it, it's great. No docs necessary, source should be reasonably simple :)

Reply:
https://github.com/facebook/jest/pull/4457 :)